### PR TITLE
feat: create command describe edge application

### PR DESCRIPTION
**Pull request by maxwelbm**
- create command describe
- create sub command edge application from describe

### CODE REVIEW

File: messages/describe/edge_applications/errors.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package edge_applications
+
+import "errors"
+
+var (
+	ErrorGetApplication       = errors.New("Failed to get the Edge Application: %s. Check your settings and try again. If the error persists, contact Azion support")
+	ErrorConvertIdApplication = errors.New("The application ID you provided is invalid. The value must be an integer. You may run the 'azion list edge-application' ommand to check your application ID")
```
Comment (helio-neto) [inline messages/describe/edge_applications/errors.go:7]: Just a small typo. The `c` letter was missing. 
```suggestion
	ErrorConvertIdApplication = errors.New("The application ID you provided is invalid. The value must be an integer. You may run the 'azion list edge-application' command to check your application ID")
```

File: messages/describe/messages.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package describe
+
+var (
+	Usage            = "describe"
+	ShortDescription = "Returns information about a specific configuration for each sub-command"
+	LongDescription  = "Returns information about a specific configuration for each sub-command, based on a given ID, in detail"
+	FlagHelp         = "Displays more information about the create command"
```
Comment (PatrickMenoti) [inline messages/describe/messages.go:7]: Displays more information about the describe command

File: pkg/cmd/delete/delete.go [hunk 3]
```diff
@@ -23,7 +21,6 @@ func NewCmd(f *cmdutil.Factory) *cobra.Command {
 	}
 
 	cmd.AddCommand(edgeApplication.NewCmd(f))
-	cmd.AddCommand(ruleEngine.NewCmd(f))
```
Comment (PatrickMenoti) [inline pkg/cmd/delete/delete.go:26]: add this back :) 
As well as delete/rules_engine.go and delete/rules_engine_test.go

File: pkg/cmd/describe/edge_applications/edge_applications.go [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+	var applicationID string
+	opts := &contracts.DescribeOptions{}
+	cmd := &cobra.Command{
+		Use:           msg.Usage,
+		Short:         msg.ShortDescription,
+		Long:          msg.LongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		Example: heredoc.Doc(`
+        $ azioncli describe edge_applications --application-id 4312
```
Comment (PatrickMenoti) [inline pkg/cmd/describe/edge_applications/edge_applications.go:29]: azion not azioncli

Comment (maxwelbm) [reply pkg/cmd/describe/edge_applications/edge_applications.go:29]: thanks man 🥇

**Review by PatrickMenoti — APPROVED** [review]

**Review by PatrickMenoti — APPROVED** [review]

**Pull request merged**